### PR TITLE
Publish Docker Image in Daily Build

### PR DESCRIPTION
**Pull request by asifdxtreme**
Partially fixes https://github.com/volcano-sh/volcano/issues/347

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -58,6 +58,16 @@ jobs:
...
           on:
             tags: true
+    - stage: Publish latest Docker Image on Daily Build
+      before_deploy:
+        - export TRAVIS_TAG=$(git describe --tags)
+      script:
+        - echo "Publish latest Docker Images to DockerHub"
+      deploy:
+        - provider: script
+          script: make TAG=${TRAVIS_TAG} RELEASE_VER=latest release
```
Comment (TommyLike) [inline .travis.yml]: We need always generate the images with `latest` tag, since it's what we used in the all-in-one yaml.

Comment (asifdxtreme) [reply .travis.yml]: got it..updated

**Comment by TommyLike** [comment]
thanks @asifdxtreme  /lgtm

File: .travis.yml [hunk 2]
```diff
@@ -58,6 +58,14 @@ jobs:
           skip_cleanup: true
           on:
             tags: true
+    - stage: Publish latest Docker Image on Daily Build
```
Comment (hzxuzhonghu) [inline .travis.yml]: How can ci control it is run once a day?

Comment (asifdxtreme) [reply .travis.yml]: `condition: $TRAVIS_EVENT_TYPE = cron` it is based on this condition
currently cron job is configured to run daily once

**Pull request merged**